assets/js/containers/VisibleEventList.jsx

```diff
@@ -0,0 +1,8 @@
+import { connect } from 'react-redux'
+import EventList from '../components/EventList'
+
+const mapStateToProps = state => state
+
+const VisibleEventList = connect(mapStateToProps)(EventList)
+
+export default VisibleEventList
```

Comment:
This basically doesn't do anything and is totally unnecessary rn but i left the scaffolding up